dictator/CommentBox.html

```diff
@@ -0,0 +1,28 @@
+{% block title %}
+    Quick Survey
+{% endblock %}
+
+{% block content %}
+<div class="card-body">
+    We appreciate feedback.
+    If you have any comments on the implementation of this study, or on the user interface, please write it down below.
+
+    {% formfield player.comment_box label='' %}
+</div>
+
+<div class="card-body">
+    Would you accept to be matched with another participant in order to be the receiver of another decider?
+    You would receive the bonus for the receiver from their decisions.
```

Comment:
For some reason, the pdf that you sent in the mail displayed this as: 
"tYou (!) would receive the bonus ..." 
However, this doesn't seem to be an issue here. Could you check what's going on?

Reply:
I think I corrected it shortly after taking the screenshots